models/iaf_psc_exp.h

```diff
@@ -34,43 +34,93 @@
-Thus, postsynaptic currents have an infinitely short rise time.
+``iaf_psc_exp``  a leaky integrate-and-fire model with
+
+* a hard threshold,
+* a fixed refractory period,
+* no adaptation mechanisms,
+* exponential shaped synaptic input currents according to [1]_
+
+Membrane potential evolution, spike emission, and refractoriness
+................................................................
+
+The membrane potential evolves according to
+
+.. math::
+
+   \frac{dV_\text{m}}{dt} = -\frac{V_{\text{m}} - E_\text{L}}{\tau_{\text{m}}} + \frac{I_{\text{syn}} + I_\text{e}}{C_{\text{m}}}
+
+where the synaptic input current :math:`I_{\text{syn}}(t)` is discussed below and :math:`I_\text{e}` is
+a constant input current set as a model parameter.
+
+A spike is emitted at time step :math:`t^*=t_{k+1}` if
+
+.. math::
+
+   V_\text{m}(t_k) < V_{th} \quad\text{and}\quad V_\text{m}(t_{k+1})\geq V_\text{th} \;.
+
+Subsequently,
+
+.. math::
+
+   V_\text{m}(t) = V_{\text{reset}} \quad\text{for}\quad t^* \leq t < t^* + t_{\text{ref}} \;,
+
+that is, the membrane potential is clamped to :math:`V_{\text{reset}}` during the refractory period.
 
-The threshold crossing is followed by an absolute refractory period (``t_ref``)
-during which the membrane potential is clamped to the resting potential
-and spiking is prohibited.
+Synaptic input
+..............
 
-The neuron dynamics is solved on the time grid given by the computation step
-size. Incoming as well as emitted spikes are forced to that grid.
+The synaptic input current has an excitatory and an inhibitory component
 
-The linear subthreshold dynamics is integrated by the Exact
-Integration scheme [2]_, which is more precise, but different from the
-implementation in [1]_, which uses the forward Euler integration scheme.
-This precludes an exact numerical reproduction of the results from [1]_.
+.. math::
 
-An additional state variable and the corresponding differential
-equation represents a piecewise constant external current.
+   I_{\text{syn}}(t) = I_{\text{syn, ex}}(t) + I_{\text{syn, in}}(t)
 
-The general framework for the consistent formulation of systems with
-neuron like dynamics interacting by point events is described in
-[2]_. A flow chart can be found in [3]_.
+where
 
-Spiking in this model can be either deterministic (delta=0) or stochastic (delta
-> 0). In the stochastic case this model implements a type of spike response
-model with escape noise [4]_.
+.. math::
+
+   I_{\text{syn, X}}(t) = \sum_{j} w_j \sum_k i_{\text{syn, X}}(t-t_j^k-d_j) \;,
+
+where :math:`j` indexes either excitatory (:math:`\text{X} = \text{ex}`)
+or inhibitory (:math:`\text{X} = \text{in}`) presynaptic neurons,
+:math:`k` indexes the spike times of neuron :math:`j`, and :math:`d_j`
+is the delay from neuron :math:`j`.
+
+The individual post-synaptic currents (PSCs) are given by
+
+.. math::
+
+   i_{\text{syn, X}}(t) = e^{-\frac{t}{\tau_{\text{syn, X}}}} \Theta(t)
```

Comment:
I would include the weight `w` here and in the following equations to ensure that units work out, in the same corresponding way as for `iaf_psc_delta`.